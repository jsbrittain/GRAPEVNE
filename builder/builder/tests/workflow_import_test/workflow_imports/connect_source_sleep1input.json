[
  {
    "id": "idcode",
    "name": "source_test",
    "type": "source",
    "config": {
      "snakefile": "builder/tests/workflow_import_test/modules/source/workflow/Snakefile",
      "params": {
        "output_filename": "mark",
        "output_namespace": "examples_sleep1"
      }
    }
  },
  {
    "id": "idcode",
    "name": "sleep1input_test",
    "type": "module",
    "config": {
      "snakefile": "builder/tests/workflow_import_test/modules/sleep1input/workflow/Snakefile",
      "params": {
        "input_filename": "mark",
        "input_namespace": "in",
        "output_filename": "mark",
        "output_namespace": "examples_sleep1",
        "sleeptime": 1
      }
    }
  },
  {
    "name": "Join [Source-to-Sleep1input]",
    "type": "connector",
    "config": {
      "map": ["source_test", "sleep1input_test"]
    }
  }
]
