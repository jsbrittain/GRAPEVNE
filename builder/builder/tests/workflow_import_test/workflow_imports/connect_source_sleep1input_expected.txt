config={}
config["input_namespace"]="None"
config["output_namespace"]="sleep1input_test"
config["sleep1input_test"]={}
config["sleep1input_test"]["config"]={}
config["sleep1input_test"]["config"]["input_filename"]="mark"
config["sleep1input_test"]["config"]["input_namespace"]="source_test"
config["sleep1input_test"]["config"]["output_filename"]="mark"
config["sleep1input_test"]["config"]["output_namespace"]="sleep1input_test"
config["sleep1input_test"]["config"]["sleeptime"]="1"
config["sleep1input_test"]["name"]="sleep1input_test"
config["sleep1input_test"]["snakefile"]="builder/tests/workflow_import_test/modules/sleep1input/workflow/Snakefile"
config["sleep1input_test"]["type"]="module"
config["source_test"]={}
config["source_test"]["config"]={}
config["source_test"]["config"]["input_namespace"]="None"
config["source_test"]["config"]["output_filename"]="mark"
config["source_test"]["config"]["output_namespace"]="source_test"
config["source_test"]["name"]="source_test"
config["source_test"]["snakefile"]="builder/tests/workflow_import_test/modules/source/workflow/Snakefile"
config["source_test"]["type"]="module"

module source_test:
    snakefile:
        config["source_test"]["snakefile"]
    config:
        config["source_test"]["config"]
use rule * from source_test as source_test_*

module sleep1input_test:
    snakefile:
        config["sleep1input_test"]["snakefile"]
    config:
        config["sleep1input_test"]["config"]
use rule * from sleep1input_test as sleep1input_test_*
